.github/workflows/test-cluster-config.yaml

```diff
@@ -0,0 +1,78 @@
+name: Test Cluster Config
+on:
+  workflow_dispatch:
+    inputs:
+      cluster_config:
+        description: 'Which cluster config type? (options are: "gpu")'
+        required: true
+      from_branch:
+        description: 'Which branch to source release branch from? (default: master)'
+        required: false
+        default: 'master'
+jobs:
+  create-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: validate version
+        run: echo "${{github.event.inputs.cluster_config}}" | grep -E '^gpu$'
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.inputs.from_branch}}
+      - name: Setup SSH Keys and known_hosts
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+        run: |
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+      - name: install go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+      - name: Install helm
+        run: |
+          curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
+          sudo apt-get install apt-transport-https --yes
+          echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+          sudo apt-get update
+          sudo apt-get install helm
+      - name: Install k
+        run: |
+          sudo curl -o /usr/local/bin/k https://raw.githubusercontent.com/jakepearson/k/master/k
+          sudo chmod +x /usr/local/bin/k
+      - name: Build aks-engine binary
+        run: make build-binary
+      - name: Validate gpu + docker scenario
+        if: ${{ github.event.inputs.cluster_config == 'gpu' }}
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+          ORCHESTRATOR_RELEASE: "1.22"
+          CLUSTER_DEFINITION: "examples/kubernetes-gpu/kubernetes.json"

```

Comment:
What do you think about taking the cluster definition file uri as a input var?

Reply:
We could do that, and we'd want to probably expose a skip and focus input var as well to run more targeted tests that are appropriate for validating particular cluster config behaviors.